README.md

```diff
@@ -4,19 +4,19 @@
 
 An extension for ReSharper and Rider IDE that highlights structured logging templates and contains some useful analyzers
 
-> [The highlighting is a built-in feature starting from R#/Rider 2021.2](https://github.com/olsh/resharper-structured-logging/issues/35#issuecomment-900883583), 
```

Comment:
My editor automatically made adjustments to whitespace. Please let me know if you would like me to revert the extraneous changes.

Reply:
Yes, please keep the whitespaces, they were made intentionally, two whitespace at the end of line is a newline character.

Reply:
I've updated the PR to only include my additions.